database/seeders/ContentSeeder.php

```diff
@@ -180,12 +200,226 @@ private function seedGenreAndCategoriesAndAreas(): void
         DB::table('areas')->insert([
             'name' => 'Außenbereich',
         ]);
-        DB::table('categories')->insert([
-            'name' => 'Festivals'
-        ]);
-        DB::table('genres')->insert([
-            'name' => 'Rock'
-        ]);
+
+        $faker = \Faker\Factory::create();
+
+        for ($i = 0; $i < 10; $i++) {
+            Category::create([
```

Comment:
SHOULD: Es ist immer sinnvoll hier Service Funktionen zu benutzen. $this->categoryService->create(). Die können einfach per constructor wie gewohnt injiziert werden

Damit ist die Erstelllogik überall gleich

(Gilt für alle Seeder)